### PR TITLE
Output valid JSON

### PR DESCRIPTION
**Pull request by goncalossilva**
At the moment, JSON output looks like this:

```
{ valid_json_object }
{ valid_json_object }
{ valid_json_object }
(...)
```

Which, on the whole, is invalid JSON.

This PR makes it valid. If the original goal was to split issues by line, the same can still be achieved, e.g.:

```
process.stdout.write("[\n");
for (const issue of issues) {
  process.stdout.write(`  ${JSON.stringify(issue)},\n`);
}
process.stdout.write("]");
```

### CODE REVIEW

**Review by Chocrates — APPROVED** [review]
Looks good to me! 🦑 

I don't recall why we did it the original way, but valid JSON probably makes sense.

**Comment by jasonmacgowan** [comment]
@goncalossilva I'm open to making this a flag, but I'd like to not change the original behavior.  The intention of the tool as written is to allow you to act on individual issues (valid JSON object per line) without having to wait for every issue to be fetched.  Writing JSON objects out as they are ready makes it easier to use this in a stream rather than having to hold all of them in memory.

Technically we are still holding all issues in memory today, but that is planned to change in the near future.

**Review by jasonmacgowan — CHANGES_REQUESTED** [review]
@goncalossilva I'm open to making this a flag, but I'd like to not change the original behavior.  The intention of the tool as written is to allow you to act on individual issues (valid JSON object per line) without having to wait for every issue to be fetched.  Writing JSON objects out as they are ready makes it easier to use this in a stream rather than having to hold all of them in memory.

Technically we are still holding all issues in memory today, but that is planned to change in the near future.

**Comment by goncalossilva** [comment]
I think that's doable. 😊

The only challenge I see is that the output is still not JSON... But it is [JSON Lines](https://jsonlines.org/). A path forward could be:
- Rename `--format JSON` flag to `--format JSONL`, keep it as default
- Add `--format JSON` to output valid JSON

Thoughts?

**Comment by jasonmacgowan** [comment]
@goncalossilva that seems reasonable to me!

File: packages/cli/lib/commands/repo/releases.js [hunk 1]
```diff
@@ -65,11 +65,14 @@ class RepoReleases extends base_1.default {
             progress.tick(results.repository.releases.nodes.length);
             releases.push(...results.repository.releases.nodes);
         } while (results.repository.releases.pageInfo.hasNextPage);
-        if (format === "JSON") {
+        if (format === "JSONL") {
```
Comment (Chocrates) [inline packages/cli/lib/commands/repo/releases.js:68]: Not something we should worry about with this one, but the fact that you had to do this a bunch of times makes me think we should refactor this 😄

**Review by Chocrates — APPROVED** [review]
Looks good to me.  I tested the GUI and it works as expected.  I was a bit surprised that JSONL saves as a `.jsonl` file, but I suppose that makes sense.
I need to figure out why the linter is angry before we can merge.

**Comment by Chocrates** [comment]
@goncalossilva can you run `npm run lint:fix` from the root of the project and commit those changes?
Got a couple prettier errors as well.  I opened another PR to fix the Super Linter error, once we get that fixed and merged your other job should magically start passing.

**Comment by goncalossilva** [comment]
> I was a bit surprised that JSONL saves as a .jsonl file, but I suppose that makes sense.

Indeed, that is [the suggestion](https://jsonlines.org/):

> JSON Lines files may be saved with the file extension .jsonl

---

> Not something we should worry about with this one, but the fact that you had to do this a bunch of times makes me think we should refactor this smile

Haha, I considered this... But as you noted, it was out of scope. It'd be great to improve this. :+1: 

> can you run npm run lint:fix from the root of the project and commit those changes?

Done, apologies for missing this before!

**Comment by Chocrates** [comment]
@jasonmacgowan if you are happy with this I say we merge it and open a defect to get the linter and formatter aligned.  I don't know why this is all of a sudden complaining.

**Pull request merged**